pyblish_lite/window.py

```diff
@@ -830,6 +831,17 @@ def on_was_processed(self, result):
             if instance.id not in models["instances"].ids:
                 models["instances"].append(instance)
 
+            family = instance.data['family']
+            if family:
+                plugins_filter = self.data['views']['right'].model()
+                plugins_filter.add_inclusion(role='families', value=family)
+
+            families = instance.data['families'] if 'families' in instance.data  else False
+            if families:
```

Comment:
I'm not seeing where this is cleared. It looks as though new inclusions are being added over and over, even on publishing different things, as the `plugin_filter` object would live for as long as the Windows is alive.

What do you think?